docs/outlook/one-outlook.md

```diff
@@ -0,0 +1,81 @@
+|Online meetings|Enable users to create and join online meetings.|Supported.|
+|Meeting enhancements|Provide additional services for users when they schedule meetings, such as location selection, catering services, and room lighting and temperature adjustments.|Supported.|
+|Online signatures|Automatically add themed signatures to messages and appointments.|Supported.|
+|Customer relationship management (CRM) and tracking services|Enable users to send and retrieve information from their CRM system to track communications with existing and potential customers.|Supported.|
+|Content reuse|Enable users to transfer and retrieve text and other content types from third-party systems.|Supported.|
+|Mail item transformation|Enable users to transform mail items into other formats.|Supported.|
+|Project management|Enable users to create and track project work items from third-party systems.|Supported.|
+|Attachment management|Enable users to import or export attachments from third-party locations.|Partially supported. Essential features are yet to be addressed to create a similar experience to the VSTO or COM add-in.|
+|Message encryption|Enable users to encrypt and decrypt messages.|Partially supported. Essential features are yet to be addressed to create a similar experience to the VSTO or COM add-in.|
+|Data loss prevention|Prevent users from forwarding mail items that contain highly sensitive information.|Partially supported. Essential features are yet to be addressed to create a similar experience to the VSTO or COM add-in.|
+|Mail item classification|Enable users to identify and classify messages that contain sensitive information.|Partially supported. Essential features are yet to be addressed to create a similar experience to the VSTO or COM add-in.|
+|Data sync service|Enable bidirectional synchronization of mail items with third-party systems.|Partially supported. Essential features are yet to be addressed to create a similar experience to the VSTO or COM add-in.|
+|Proofing mail items|Provide users with real-time proofreading assistance as they compose messages.|Not currently supported.|
+
+There are various possibilities to extend the Outlook functionality through add-ins. If your VSTO or COM add-in solution doesn't quite fit any of the scenarios in the table, [complete the survey to share your scenario](https://aka.ms/DevNewOutlook).
+
+## Support for classic Outlook on Windows
+
+The classic Outlook on Windows desktop client will continue to support the development of new and existing Outlook web add-ins. Additionally, it will continue to receive releases of the latest Outlook add-in features.
+
+## Test your add-in in the new Outlook on Windows
+
+Test your Outlook web add-in in the new Outlook on Windows today! To switch to the new Outlook on Windows that's in preview, you must:
+
+- Have an Exchange-backed Microsoft 365 work or school account. The new client doesn't currently support on-premises, hybrid, or sovereign Exchange accounts.
```

Comment:
Does this mean then that, for now, Exchange Online is needed, not just Exchange?
```suggestion
- Have a Microsoft 365 work or school account connected to Exchange Online. The new client doesn't currently support on-premises, hybrid, or sovereign Exchange accounts.
```